lib/rubycritic/core/rating.rb

```diff
@@ -22,6 +22,16 @@ def to_h
       @letter
     end
 
+    def color
+      case @letter
+      when "A" then :green
```

Comment:
Symbols based on colorize; could refactor the html reporter to switch based on the symbol if we wanted